src/WebJobs.Script/Description/DotNet/DotNetFunctionDescriptorProvider.cs

```diff
@@ -156,14 +156,6 @@ protected override IFunctionInvoker CreateFunctionInvoker(string scriptFilePath,
                     descriptors.Add(new ParameterDescriptor(ScriptConstants.SystemExecutionContextParameterName, typeof(ExecutionContext)));
                 }
 
-                // If we have an HTTP trigger binding but no parameter binds to the raw HttpRequestMessage,
-                // add it as a system parameter so it is accessible later in the pipeline.
-                if (string.Compare(triggerMetadata.Type, "httptrigger", StringComparison.OrdinalIgnoreCase) == 0 &&
```

Comment:
This isn't related to http response / return. How are you able to remove this?

Reply:
The fix for this https://github.com/Azure/azure-webjobs-sdk-extensions/issues/273 solves that.  So I should be able to remove the code you linked to too (once we have an updated Http extension). I'll verify.

Reply:
Yes, let's remove that code as well then, once you verify.